robot/impl/local_robot.go

```diff
@@ -38,6 +39,10 @@ import (
 
 var _ = robot.LocalRobot(&localRobot{})
 
+// ResourceConfigurationTimeout sets how long a constructor or reconfigure call should
+// wait before timing out.
+var ResourceConfigurationTimeout = time.Minute
```

Comment:
If others agree on potentially not leaving this exported as a knob, I wonder if we should unexport this variable and move your test that modifies it to `robot_reconfigure_test.go`. That file is in the `robotimpl` package (unlike `local_robot_test.go` which is in `robotimpl_test`) and will be able to modify the private value.